docs/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Home
+---
+
+This pipeline uses Centrifuge for novel microbial classification and enables rapid,
```

Comment:
Novel? Centrifuge is also quite old already :wink:.

Reply:
I took that line from the Centrifuge documentation :stuck_out_tongue_winking_eye:. But I can change it of course.